### PR TITLE
Created Issue Template. Initial commit

### LINKED ISSUE #1: Add Issue Template

### PR DESCRIPTION
**Pull request by MASTERCHIEF011**
Fixes #1

### CODE REVIEW

**Comment by MASTERCHIEF011** [comment]
@YashKumarVerma  can u pls verify whether we have to do it like this. I will add the other modules likewise after u have verified.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issue_enabled: true
+contact_links:
+  - name: Email Module
+    url: https://github.com/UniAuth/uniauth-backend/tree/master/src/mailer
+    about: This Module handles the email service of UniAuth
```
Comment (YashKumarVerma) [inline .github/ISSUE_TEMPLATE/config.yml:5]: what is this ?

File: .github/ISSUE_TEMPLATE/email-issue.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+name: 'Email Module Issue'
+about: 'Open a new issue'
+titile: '[Request]'
+labels: ''
+assignees: ''
+---Please describe your issue.
+
+### Examples
+
+Please provide an example image of the issue
```
Comment (YashKumarVerma) [inline .github/ISSUE_TEMPLATE/email-issue.md:11]: this is not github action, read up sample templates, please.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/email-issue.md (hunk 2 above)
Comment (MASTERCHIEF011) [reply .github/ISSUE_TEMPLATE/email-issue.md:11]: Okay will check them out

**Pull request closed without merging**